src/treeview/TreeView.jsx

```diff
@@ -39,7 +32,7 @@ function TreeNodes({ childrenPath, depth, draggable, nodes, parentNode, template
           childrenPath={childrenPath}
           depth={depth}
           draggable={draggable}
-          relativePath={`${relativePath}[${index}]`}
+          relativePath={relativePath ? `${relativePath}[${index}]` : `[${index}]`}
```

Comment:
There was a misunderstanding in the last PR, as the nodes are already an array the correct path to the elements in the first level of recursion should be only the index.


Before change:
![image](https://user-images.githubusercontent.com/29656850/115390988-4db82b80-a1b5-11eb-86fe-2843b2245c3a.png)

After change: 
![image](https://user-images.githubusercontent.com/29656850/115391103-75a78f00-a1b5-11eb-93b6-0cfa218e0a06.png)

Reply:
![image](https://user-images.githubusercontent.com/29656850/115391193-9243c700-a1b5-11eb-95d0-648bf4121735.png)